yt_dlp/extractor/bilibili.py

```diff
@@ -535,6 +540,84 @@ def _real_extract(self, url):
         return self.playlist_result(self._entries(list_id), list_id)
 
 
+# https://www.bilibili.com/v/kichiku/mad/#/
+class BilibiliCategoryIE(SearchInfoExtractor):
```

Comment:
Why inherit from `SearchInfoExtractor`? This is behaving like a normal `InfoExtractor`

Reply:
that's fair; changed